python/ray/tests/test_unavailable_actors.py

```diff
@@ -69,16 +95,18 @@ def sigkill_actor(actor):
     "caller",
     ["actor", "task", "driver"],
 )
-def test_actor_unavailable_conn_broken(ray_start_regular, caller):
+def test_actor_unavailable_conn_broken(ray_start_regular_with_patch, caller):
```

Comment:
You can just do 

```
@pytest.mark.parametrize("ray_start_regular", [{"log_to_driver": False}], indirect=True)
def test_actor_unavailable_conn_broken(ray_start_regular):
```

No need to create a new fixture. Also we can just simply disable it for both linux and mac.

Reply:
I found that we can really solve this by adding a new RPC. Please review again.